README.md

```diff
@@ -20,7 +20,7 @@ React Router keeps your UI in sync with the URL. It has a simple API with powerf
 
 **Older Versions:**
 
-- 0.13.x - [docs](https://github.com/reactjs/react-router/tree/0.13.x/docs/guides) / [code](https://github.com/reactjs/react-router/tree/0.13.x) / [upgrade guide](https://github.com/reactjs/react-router/blob/master/upgrade-guides/v1.0.0.md)
+- 0.13.x - [docs](https://github.com/reactjs/react-router/tree/0.13.x/doc) / [code](https://github.com/reactjs/react-router/tree/0.13.x) / [upgrade guide](https://github.com/reactjs/react-router/blob/master/upgrade-guides/v1.0.0.md)
```

Comment:
Afraid so :(

Could also update the 0.13.x branch so `doc` and `docs` are consolidated. That way guides are in the same place as API doc.